src/Traits/MagicPropertyTrait.php

```diff
@@ -19,6 +19,32 @@
 /**
  * Provides the magic methods that allow read access
  * to magic properties.
+ *
+ * @property-read int year
```

Comment:
Arent those usually variables? $year etc?
See the other annotations in other files.